scripts/render-notebooks.R

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env Rscript
+#
+# Make rendered versions of .Rmd files with automagic citations with a specified
+# .bib file.
+
+# Find the project root
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+
+# List of files to render
+infiles <- c(
+  file.path(root_dir, "01-getting-started", "getting-started.Rmd")
+)
+
+# Load library:
+library(optparse)
+
+# Set up optparse options
+option_list <- list(
+  make_option(
+    opt_str = c("-b", "--bib_file"), type = "character",
+    default = "references.bib", # Default is this file, but it can be changed
+    help = "File path relative to root directory (where .git is)
```

Comment:
I don't mind having the files being processed one at a time, but it does concern me that a file might be changed and accidentally not re-rendered with this script. 

Is there a non-janky way to optionally re-render only files that have had edits done to them?

Reply:
snakemake? (ducks)

Reply:
To be honest, this is actually a good argument for a non-monolithic script (one that works on one file at a time). And really it is a pretty good place to start writing `snakemake`! This would be a one step workflow: you would have a `main` rule that defines the output files, and a `render` rule that looks for the appropriate input file to generate those output files then runs the appropriate shell script.

Reply:
You've convinced me.